### PR TITLE
Re-design User Authentication Page

### PR DESCRIPTION
**Pull request by clintfidel**
#### What does this PR do?
- This pull request re-design user authentication 

#### Description of Task to be completed?
- Re design user sign up page
- Re design user login page

#### How should this be manually tested?
- visit application url [here](https://asgard-cfh-staging.herokuapp.com)

#### What are the relevant pivotal tracker stories?
- [157801297](https://www.pivotaltracker.com/story/show/157801297)

#### Screenshots (if appropriate)
screen shot 2018-06-12 at 1 21 41 pm

### CODE REVIEW

File: app/views/includes/foot.jade [hunk 1]
```diff
@@ -8,6 +8,8 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 
 //Bootstrap
 script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.js')
```
Comment (fortunee) [inline app/views/includes/foot.jade]: Hi, @clintfidel seems like you've added another `boostrap.js` on line  22, please get rid of one of them.

File: public/views/index.html [hunk 2]
```diff
@@ -300,3 +296,159 @@ <h3>XOXO</h3>
 </section>
 <!-- Footer End -->
 </div>
+
+<!-- New landing page -->
+<!--
```
Comment (fortunee) [inline public/views/index.html]: Let's get rid of this large chunk of commented out code below @clintfidel @saheedt

**Review by fortunee — COMMENTED** [review]
Hi, @clintfidel @saheedt Please check out the comments and refactor.

**Comment by clintfidel** [comment]
Noted @fortune, feedback would be implemented asap

**Comment by clintfidel** [comment]
Hi @fortunee feeback implemented

**Review by fortunee — APPROVED** [review]
This looks good now @clintfidel @saheedt Thanks for the good work 👍

**Pull request merged**